.github/workflows/mirror-php-package.yml

```
@@ -0,0 +1,36 @@
+    mirror:
+        name: Run packages release script
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        steps:
+            - name: Checkout monorepo
+              uses: actions/checkout@v4
+
+            - name: Setup WooCommerce Monorepo
+              uses: ./.github/actions/setup-woocommerce-monorepo
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)